crates/frontend/src/pages/context_override.rs

```diff
@@ -151,7 +151,25 @@ pub fn context_override() -> impl IntoView {
 
     let handle_context_create = Callback::new(move |_| {
         set_form_mode.set(Some(FormMode::Create));
-        set_selected_data.set(None);
+        let PageResource {
+            config: _,
+            dimensions,
+            default_config: _,
+        } = page_resource.get().unwrap_or_default();
+        let context_with_mandatory_dimensions = dimensions
+            .into_iter()
+            .filter_map(|dim| {
+                if dim.mandatory {
+                    Some((dim.dimension, String::from(""), String::from("")))
```

Comment:
if the 2nd and 3rd argument of the tuple are always empty string, do we at all need these 2 params of the tuple ?

Reply:
Yes because its vec<(String,String, String)>, dimension, operator and value.